consai2r2_teleop:
  ros__parameters:
    has_analog_d_pad        : false
    d_pad_up                : 13
    d_pad_down              : 14
    d_pad_left              : 15
    d_pad_right             : 16
    # for analog_d_pad 
    d_pad_up_is_positive    : true
    d_pad_right_is_positive : false

    button_shutdown_1       : 8
    button_shutdown_2       : 9
    button_color_id_enable  : 6

    button_move_enable      : 4
    axis_vel_surge          : 1
    axis_vel_sway           : 0
    axis_vel_angular        : 3

    button_kick_enable      : 5
    button_kick_straight    : 3
    button_kick_chip        : 2

    button_dribble_enable   : 7

    d_pad_change_color      : "right"
    d_pad_increment         : "up"
    d_pad_decrement         : "down"
    d_pad_reset             : "right"